docs/application/tizen-studio/rt-ide/overview.md

```diff
@@ -1,24 +1,15 @@
-# Tizen RT
+# TizenRT
 
-Tizen has commercialized in smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of Tizen RT is to extend the Tizen platform device coverage to these kind of low-end devices.
+Tizen platform is commercialized on smart TV, smartphone, wearable devices (Gear S, Gear Fit), and smart home appliances. However, low-end and low-cost IoT devices, such as home appliances without display and wearable bands with a small LCD, have received less attention. The objective of TizenRT is to extend the Tizen platform device coverage to these kind of low-end devices.
```

Comment:
Would you please re-check the update? That I can't see the fix in below communication.


![image](https://user-images.githubusercontent.com/9958374/48320959-2ebb2400-e662-11e8-8ef5-1448af24d9d3.png)

Reply:
i fixed.